fastlane/lib/fastlane/documentation/markdown_docs_generator.rb

```diff
@@ -43,6 +43,40 @@ def all_actions_from_enhancer
       @_launches ||= JSON.parse(File.read(File.join(Fastlane::ROOT, "assets/action_ranking.json"))) # root because we're in a temporary directory here
     end
 
+    def actions_path
+      "lib/fastlane/actions/"
+    end
+
+    def where_is(klass)
```

Comment:
what does `klass` stand for?

Reply:
Its the `class` of the action but `class` is a reserved word so we have to go with `klass` 😇

Reply:
omg